pyvista/plotting/plotting.py

```diff
@@ -1554,6 +1564,27 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
 
         ##### Plot a single PyVista mesh #####
 
+        silhouette_params = dict(rcParams['silhouette'])
+        if isinstance(silhouette, dict):
+            silhouette_params.update(silhouette)
+            silhouette = True
+        if silhouette:
```

Comment:
Again bikeshedding, but if we're going to accept truthy non-dict values anyway, we could just keep `silhouette` as a dict and re-enter the second block with that. After all if someone passes an empty dict for `silhouette` then arguably they don't want silhouettes (and in any other case `if silhouette` would be entered). But I might be wrong.
```suggestion
        if isinstance(silhouette, dict):
            silhouette_params.update(silhouette)
        if silhouette:
```

Reply:
Not sure about this one... Empty `dict` would become a weird way to disable silhouette in my opinion. The way I see it, you use a dict to specify values different from the default so if it's empty, you should use all default values. At least, this is what I would expect.